Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationOverview.cshtml.cs

```diff
@@ -19,7 +19,7 @@ public async Task<IActionResult> OnGet()
     {
         try
         {
-            OrganisationDetails = await organisationClient.GetOrganisationAsync(Id);
+            OrganisationDetails = await organisationClient.GetOrganisationExtendedAsync(Id);
```

Comment:
Had to call a new "extended" endpoint here, since we want details about the review status of the org, not just the basic fields.

Reply:
discussed: let's call the new `GET /organisations/{organisationId}/approvals` here to enrich organisation details if it has pending roles